core/domain/topic_jobs_one_off.py

```diff
@@ -92,3 +95,64 @@ def reduce(key, values):
                 sum(ast.literal_eval(v) for v in values))])
         else:
             yield (key, values)
+
+
+class PopulateTopicThumbnailSizeOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job to populate the thumbnail_size_in_bytes attribute in topic
+     models.
+     """
+
+    _DELETED_KEY = 'topic_deleted'
+    _ERROR_KEY = 'thumbnail_size_update_error'
+    _EXISTING_SUCCESS_KEY = 'thumbnail_size_already_exists'
+    _NEW_SUCCESS_KEY = 'thumbnail_size_newly_added'
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [topic_models.TopicModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            yield PopulateTopicThumbnailSizeOneOffJob._DELETED_KEY, 1
+            return
+
+        topic_model = topic_models.TopicModel.get(item.id)
```

Comment:
In this job, `item` is the `TopicModel`, right? No need to 'get' it again.

Reply:
Oh, thanks for the inputs. Noted.

Reply:
Updated.